src/pages/db/index.astro

```diff
@@ -32,7 +33,7 @@ const description = "The SQL database platform for content-driven websites."
 	</div>
 </MainLayout>
 
-<style>
+<style is:global>
 	.db {
 		background-color: #060913 !important;
 	}
```

Comment:
I'd definitely rather pass this as a dynamic class name than make it global and `!important` 😂

Reply:
I swear there was a good reason to have the style block here at one point, I just totally forgot to remove it now that there's only a single style 😂